README.md

```diff
@@ -173,6 +173,24 @@ example for crudActionsAfter slot:
  */
 ```
 
+```
+/**
+ * rowsPerPageItems
+ * Multiple row selecting per page
```

Comment:
I'd rename this to `rowsPerPageOptions` and then the description to: "The rows per page option selection"

Reply:
Keep it the same as vuetify?

Reply:
Ah didn't know! Lets keep it that way then :)